agent.go

```diff
@@ -1210,3 +1237,42 @@ func winsizeCb(pod *pod, data []byte) error {
 
 	return nil
 }
+
+// Get the agent configuration from kenrel cmdline
+func getConfig() error {
+	kernelCmdline, err := ioutil.ReadFile(kernelCmdlineFile)
+	if err != nil {
+		return err
+	}
+	if err := parseKernelCmdline(string(kernelCmdline)); err != nil {
+		return err
+	}
+	return nil
+}
+
+func parseKernelCmdline(cmdline string) error {
+	logLevel := ""
+	words := strings.Fields(cmdline)
+	for _, w := range words {
+		if strings.HasPrefix(string(w), "agent.debug=") {
+			logLevel = strings.Split(string(w), "agent.debug=")[1]
```

Comment:
I'd create a variable rather than calling `string()` twice:

```
str := string(w)
```

Also, I think you should check the number of fields returned by `Split()` before de-referencing the 2nd element.